packages/cf.js/API_REFERENCE.md

```diff
@@ -0,0 +1,54 @@
+# API V0.0.2
+## `cf.js`
+
+- `Client`
+    - Properties
+        - `nodeProvider`
+    - Instance methods
+        - `getAppInfos(): AppInfo[]`
+        - `createAppFactory(appDefinition: AppDefinition): AppFactory`
+    - Client lifecycle
+        - `on(eventType, callback)`
+            - EventType: proposeInstall, install, rejectInstall
+- `AppFactory`
+    - Properties
+        - `appDefinition: AppDefinition`
+    - Instance methods
+        - `async proposeInstall(peerAddress, terms): Promise<AppID>`
+        - `async install(appId: AppID): Promise<AppInstance>`
+- `AppInstance`
+    - Properties
+        - `id: AppID` — Identifier for this specific app instance
+        - `definition: AppDefinition`
+    - Instance methods
+        - `async applyAction(action): AppState`
+            - `action`: JSON-encoded object congruent with AppAction struct
+            - Returns ABI decoded representation of the latest signed state of the app.
+        - `async proposeState(state)`: Propose a state to countersign
+        - `async uninstall()`
+            - Uninstall the app
+        - `async getTerms(): [assetType, limit, tokenAddress]`
```

Comment:
As an alternative to getting the terms asynchronously after the app's instantiation, could we just provide the terms (along with other app instance properties like `AppId`) in the `onInstall` notification? @cf19drofxots